src/lib/insights/listener.tsx

```diff
@@ -58,7 +58,9 @@ const insightsListener = (BaseComponent: any) => {
           const payload = parseInsightsEvent(targetWithEvent);
           props.sendEvent(payload);
         }
-      } else {
+      }
+
+      {
```

Comment:
`prop.sendEvent` exists, so it always followed the first if branch. We should run them both.

Reply:
if we should run them both, you should probably remove the curly brace here and on line 72